### ISSUE TITLE
Support multilined output

### ISSUE THREAD

**Issue by clarkwang** [webdev-sh/javascript-minifier.com#4]
Is it possible to output the minified js in multiline format so it's a bit more convenient to view and update in a text editor?

**Comment by chilts** [comment]
It's not really designed to be edited after being minified. Best to view, edit and update the original prior to minification. :)

Thanks for your feedback and let me know if there is anything else you need help with.

**Comment by clarkwang** [comment]
It's like on the https://javascript-minifier.com/ page, the "Minified Output" displays the resulted js in multiline (actually wrapped) format. I guess that's because multiline is more human friendly.

Background: I started playing with https://github.com/brookhong/Surfingkeys recently which uses js script as its configuration but there's a limit for the js size. It's cool to find https://javascript-minifier.com/ and it really helped. The only issue is after I copied the single-lined minified js (3k chars) into its conf it does not look good.

Yes it's only cosmetic thing. It's ok if the effort is not worth it.

Thanks.
